src/main/java/com/garannodou/role/controller/GoogleAuthRequest.java

```diff
@@ -10,70 +10,188 @@
 import javax.validation.constraints.Pattern;
 import javax.validation.constraints.Size;
 
-@Getter
-@Setter
+
 public class GoogleAuthRequest {
 
-    @Valid
-    private GoogleBasicProfile googleBasicProfile = new GoogleBasicProfile();
-    private GoogleAuth googleAuth = new GoogleAuth();
+    @Getter(onMethod = @__({@Valid}))
+    @Setter(onParam = @__({@Valid}))
+    protected GoogleBasicProfile googleBasicProfile = new GoogleBasicProfile();
+    @Getter(onMethod = @__({@Valid}))
+    @Setter(onParam = @__({@Valid}))
+    protected GoogleAuth googleAuth = new GoogleAuth();
 
     public GoogleAuthRequest() {
     }
 
-    @Getter
-    @Setter
-    class GoogleBasicProfile {
-        @NotNull
-        @Pattern(regexp = "^[a-zA-Z0-9-_\\s]+$",
-                message = "Name has invalid format. Expected format: ^[a-zA-Z0-9-_\\s]+$")
-        @Size(min = 10, max = 50)
+    public class GoogleBasicProfile {
+        @Getter(onMethod =
+        @__({
+                @NotNull,
+                @Pattern(regexp = "^[a-zA-Z0-9-_\\s]+$",
+                        message = "Name has invalid format. Expected format: ^[a-zA-Z0-9-_\\s]+$"),
+                @Size(min = 10, max = 50)
+        })
+        )
+        @Setter(onParam =
+        @__({
+                @NotNull,
+                @Pattern(regexp = "^[a-zA-Z0-9-_\\s]+$",
+                        message = "Name has invalid format. Expected format: ^[a-zA-Z0-9-_\\s]+$"),
+                @Size(min = 10, max = 50)
```

Comment:
We aggreed to put `@Size(min = 1, max = 150)`. See [comment](https://github.com/garan-no-dou/role-ws/pull/20#discussion_r491670467)

Reply:
Also remove the validation annotations from Lombok's annotation, it's really verbose. Better to add the validation annotations on the class field.